### DISCUSSION TITLE
Silent Install?

### DISCUSSION THREAD

**Discussion by rodw-wilkins** [microsoft/RdpGamepad#18] [General]
Hi, I am trying to include this in my build project is there a silent install option with the installer?  Like /qn or /s or /silent parameters?   I'd like the client/receiver and virtual gamepad driver on all machines via script so any of them could connect to each other for this purpose after installation without having to install all 3 manually.

**Comment by jpflouret** [comment]
/silent should work according to the [Inno Setup Documentation](https://jrsoftware.org/ishelp/index.php?topic=setupcmdline).

**Reply by rodw-wilkins** [reply]
Thanks!  Thought I tried this already but I checked again and it worked, might have had the syntax wrong or something.  Anyway these are great utilities, thanks for this works like a charm for what I need.
